pages/wishlist.html

```diff
@@ -0,0 +1,275 @@
+                        <a class="link-none" href=""> Remove From Wishlist </a>
+                    </button>
+                </div>
+            </div>
+            <div class="card card-vertical">
+                <div class="badge-content">
+                    TRENDING
+                </div>
+                <div class="card-icon-action card-badge">
+                    <a class="link-none" href="#">
+                        <i class="fas fa-heart wish-icon" aria-hidden="true"></i></a>
+                </div>
+                <img loading="lazy"
+                    src="https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_1024/nusnyjv4vv1teca0txre"
+                    alt="Pickle by Jhaji" class="card-media-img  p-0" />
+                <div class="card-content">
+                    <h4 class="txt-bold card-details-title">Proteinated Milk</h4>
+                    <p class="card-details-subtitle">By Humpy A2</p>
+                    <p class="card-pricing">
+                        <span class="card-price">₹600</span>
+                        <del class="card-original-price">₹1000</del>
+                        <span class="card-discount">40% off</span>
+                    </p>
+                </div>
+                <div class="card-actions">
+                    <button class="btn btn-warning btn-warning-hover">ADD TO CART</button>
+                    <button class="btn btn-outline">
+                        <a class="link-none" href=""> Remove From Wishlist </a>
+                    </button>
+                </div>
+            </div>
+            <div class="card card-vertical">
+                <div class="badge-content">
+                    TRENDING
+                </div>
+                <div class="card-icon-action card-badge">
+                    <a class="link-none" href="#">
+                        <i class="fas fa-heart wish-icon" aria-hidden="true"></i></a>
+                </div>
+                <img loading="lazy"
+                    src="https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_1024/nusnyjv4vv1teca0txre"
+                    alt="Pickle by Jhaji" class="card-media-img  p-0" />
+                <div class="card-content">
+                    <h4 class="txt-bold card-details-title">Proteinated Milk</h4>
+                    <p class="card-details-subtitle">By Humpy A2</p>
+                    <p class="card-pricing">
+                        <span class="card-price">₹600</span>
+                        <del class="card-original-price">₹1000</del>
+                        <span class="card-discount">40% off</span>
+                    </p>
+                </div>
+                <div class="card-actions">
+                    <button class="btn btn-warning btn-warning-hover">ADD TO CART</button>
+                    <button class="btn btn-outline">
+                        <a class="link-none" href=""> Remove From Wishlist </a>
```

Comment:
use the same case...ADD TO CART & Remove From Wishlist

Reply:
Same case meaning both shall be of same type?

Reply:
uppercase or lowercase

Reply:
Got it!!